demos/SSHRemoting/README.md

```diff
@@ -152,10 +168,10 @@ GitCommitId                    v6.0.0-alpha.17
 ```
 
 ### Known Issues:
-1.  You can currently establish a connection either interactively with user name and password or via key authentication.  PSCredential is not yet supported.
-2.  Remote debugging from Linux to Windows does not work.  However, remote debugging from Windows to Linux does work.
-3.  Fan out to multiple machines not yet supported.
-4.  sudo command does not work in remote session to Linux machine.
+1. You can currently establish a connection either interactively with user name and password or via key authentication.  
+2. Remote debugging from Linux to Windows does not work.  However, remote debugging from Windows to Linux does work.
```

Comment:
This can be removed.  Remote debugging works in both directions (actually has for a while now).